data/json/items/tools.json

```diff
@@ -10797,5 +10797,30 @@
     "ammo": "NULL",
     "revert_to": "null",
     "use_action": "LADDER"
+  },
+  {
+    "id": "iceaxe",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "dark_gray",
+    "name": "ice axe",
+    "description": "This is an ice axe with hammer on its head, a multi-purpose hiking and climbing tool used by mountaineers.  It is sturdy enough, so you can use it to pry open closed doors or lift manhole covers.",
+    "price": 5000,
+    "material": "steel",
+    "weight": 500,
+    "volume": 3,
+    "bashing": 14,
+    "cutting": 3,
+    "flags" : ["STAB", "DURABLE_MELEE"],
```

Comment:
Well, it have a narrow, pointy end, like the one a pickaxe have.
![371px-icetool](https://cloud.githubusercontent.com/assets/11132525/9924458/60ef9ac8-5d08-11e5-80b5-d78a938c098f.jpg)
It doesn't have cutting edge to chop.